### DISCUSSION TITLE
Beta Progress of TeaClient

### DISCUSSION THREAD

**Discussion by eveeifyeve** [TeaClientMC/.github#2] [Announcements]
> [!NOTE]
> As this is work in progress this will automatically update when a new update is avalible.

Last update: 4:25am AEDT Thu 8th August 2025 (Updated website url)

# Progress of TeaClient

## Developmental stuff
- [ ] Our Discord bot (In progress @Eveeifyeve)
- [ ] Finish rust obfuscation tool for TeaClient (In progress @Eveeifyeve)
- [ ] Moderation System (In progress @swiffyshit & @Eveeifyeve)
- [ ] Website
	- [ ] Store (Requires API work TODO @Eveeifyeve) 	
	- [x] Policies (Need review by a laywer, @Eveeifyeve @unrayed)
- [x] Business: Sole-trader
	- [x] Website
	- [ ] Social medias (Discord done, Linkdein (Locked down I need connections), Twitter left)
	- [x] Registration (Waiting on business name to be checked!)
	- [x] reveal
- [ ] Client
	- [ ] Finish the splash screen, 
	- [ ] Finish title screen 
	- [ ] Finish menu gui. (In progress @Eveeifyeve)
- [ ] Launcher
	- [x] Finish servers page
	- [x] Minecraft Authentfication
	- [ ] Finish Settings (ready to pickup)
	- [ ] Finish the launching part of the launcher (wip @Eveeifyeve)

# Design-mental stuff
- [ ] Design 4 cosmetic designs for TeaClient (DUE 31 SEP)
	- [ ] Texturing (TODO @unrayed)

Estimate on our website: https://teaclient.com
This progress is to help you get an idea of how long is TeaClient beta. 

CC @TeaClientMC/client & @TeaClientMC/design

**Comment by eveeifyeve** [comment]


**Reply by MrFast-js** [reply]
ok
